### PR TITLE
Make the RPMs more rpmlint compliant.

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
the warnings comes from 1st reboot after AII install. it appears cdp-listend starts before NetworkManager has an IP


```
May 10 08:51:23 node2806 /usr/sbin/cdp-listend[1461]: Starting /usr/sbin/cdp-listend with configuration file /etc/cdp-listend.conf
May 10 08:51:23 node2806 /usr/sbin/cdp-listend[1461]: Exiting: failed to retrieve addr from gethostbyname(node2806.banette.os): ec 2
May 10 08:51:27 node2806 dhclient[1381]: DHCPDISCOVER on eth0 to 255.255.255.255 port 67 interval 8 (xid=0x101b40bf)
May 10 08:51:27 node2806 dhclient[1381]: DHCPOFFER from 10.141.10.250
May 10 08:51:27 node2806 dhclient[1381]: DHCPREQUEST on eth0 to 255.255.255.255 port 67 (xid=0x101b40bf)
May 10 08:51:27 node2806 dhclient[1381]: DHCPACK from 10.141.10.250 (xid=0x101b40bf)
```

**Comment by jrha** [comment]
This doesn't just fix rpmlint, it adds a reload to the init script... does cdp-listend even deal with HUPs correctly?

**Comment by stdweird** [comment]
@jrha nope. i'll remove it again, or do a restart on reload

**Comment by stdweird** [comment]
@jrha reload removed

File: src/main/daemon/cdp-listend [hunk 1]
```diff
@@ -16,60 +16,60 @@ SOname=`uname`
...
+    status)
+        status $LISTEN
+        RETVAL=$?
+        ;;
+    restart)
+        stop
+        start
+        ;;
+    *)
+        echo "Usage: $0 {start|stop|restart|reload|status}"
```
Comment (ned21) [inline src/main/daemon/cdp-listend]: Usage has reload but it does not appear to in the case statement?

**Comment by stdweird** [comment]
the original rpmlint warings/errors
```
cdp-listend.noarch: W: spelling-error %description -l C deamon -> daemon
cdp-listend.noarch: W: non-standard-group Quattor
cdp-listend.noarch: E: no-changelogname-tag
cdp-listend.noarch: W: invalid-license http://www.opensource.org/licenses/apache2.0
cdp-listend.noarch: W: invalid-url URL: http://quattor.org/cfg-modules/cdp-listend/ HTTP Error 404: Not Found
cdp-listend.noarch: E: useless-provides cdp-listend
cdp-listend.noarch: W: no-manual-page-for-binary cdp-listend
cdp-listend.noarch: W: no-manual-page-for-binary check-cdp-listend
cdp-listend.noarch: W: non-standard-dir-in-usr quattor
cdp-listend.noarch: E: init-script-without-chkconfig-preun /etc/init.d/cdp-listend
cdp-listend.noarch: W: service-default-enabled /etc/init.d/cdp-listend
cdp-listend.noarch: W: no-reload-entry /etc/init.d/cdp-listend
```

now it gives
```
cdp-listend.noarch: W: non-standard-group Quattor
cdp-listend.noarch: W: no-manual-page-for-binary cdp-listend
cdp-listend.noarch: W: no-manual-page-for-binary check-cdp-listend
cdp-listend.noarch: W: non-standard-dir-in-usr quattor
cdp-listend.noarch: W: service-default-enabled /etc/init.d/cdp-listend
cdp-listend.noarch: W: no-reload-entry /etc/init.d/cdp-listend
1 packages and 0 specfiles checked; 0 errors, 6 warnings.
```

the reload entry was a simple warning, but it doesn't have some simple solution

**Pull request merged**